tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -743,6 +743,27 @@ def search_versions(filter_string):
         assert exception_context.exception.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
         assert "expected a non-empty list of string values" in exception_context.exception.message
 
+        # search using an ill-formed IN operator correctly throws exception
+        with self.assertRaises(MlflowException) as exception_context:
```

Comment:
could have used @ parametrize but no need to change now.

Reply:
Ah right - yeah will keep that in mind for next time as well